wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_streamfield.scss

```diff
@@ -57,52 +72,60 @@ li.sequence-member {
     .sequence-member-inner {
         @include clearfix;
         position: relative;
-        padding: 1.5em 50px;
+        padding: 20px 30px 50px 20px;
 
         .sequence-member {
-            border: 0;
+            // Second level nested Streamfield
+            border: 1px solid $color-teal;
+            border-bottom: 1px transparent;
+
+            &:first-child {
+                border-top-right-radius: 3px;
+                border-top-left-radius: 3px;
+            }
+
+            &:last-child {
+                border-bottom: 1px solid $color-teal;
+                border-bottom-right-radius: 3px;
+                border-bottom-left-radius: 3px;
+            }
+
+            .sequence-member-inner .sequence-member {
+                // Third level nested Streamfield
+                border-color: $color-grey-3;
+            }
+
+            .sequence-member-inner .sequence-member .sequence-member-inner .sequence-member {
+                // Fourth level nested Streamfield
+                border-color: $color-teal;
+            }
```

Comment:
Could those styles differentiating between levels be moved outside of this nested declaration, so they are all side by side and we can look at them separately from other "nested Streamfield" styles?